packages/sanity/src/desk/structureBuilder/ChildResolver.ts

```diff
@@ -28,5 +29,6 @@ export interface ChildResolver {
     | ItemChild
     | Promise<ItemChild>
     | ChildObservable
+    | Observable<ItemChild>
```

Comment:
Enough for it not to allow `document.listenQuery.pipe()`as child at compile-time even though it works runtime.

Reply:
I don't quite understand, but sounds like it solves a TS error?

Reply:
Yes, allows Observable from rxjs to be passed to `.child`. I _think_ `ChildObservable` is meant to be a shim for that type maybe, but perhaps rxjs version changed or something. I have not digged into exactly what would have to be done for `Observable` to be assignable to `ChildObservable`.

Since structurebuilder lives in `sanity` we have access to these types now, so I see no reason not to simply use from `rxjs`directly.